src/Chapter1.hs

```diff
@@ -620,7 +630,12 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Integral a => a -> a
+sumLast2 n = digitA + digitB
+  where
+    digitA = x `mod` 10
+    digitB = (x `div` 10) `mod` 10
```

Comment:
Great one!
Also, I notice that you use

```haskell
x `mod` 10
```
and 
```haskell
x `div` 10
```

One hint to make your solution shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!

So you could write it this way:

```haskell
(a, b) = x `divMod` 10
```

You can see how we could pattern match on the pair 🙂